src/app/api/quality_matrix/quality.py

```diff
@@ -0,0 +1,14 @@
+import uuid
+
+from app.api.quality_matrix.collections import collection_quality
+from app.api.quality_matrix.models import Forms
+from app.api.quality_matrix.replication_source import source_quality
+
+
+async def quality(form: Forms, node_id: str):
+    if form == Forms.REPLICATION_SOURCE:
+        _quality, total = await source_quality(uuid.UUID(node_id))
```

Comment:
return directly?

Having an extra file only for this function seems a little excessive

Reply:
Refactored back to api, for now.